.github/workflows/test.yml

```diff
@@ -84,12 +84,13 @@ jobs:
       - name: cmake
         uses: jwlawson/actions-setup-cmake@v2
         with:
-          cmake-version: '3.31.x'
+          cmake-version: "3.31.x"
 
-      - name: install build dependencies (macos-latest, wasmer_wamr)
-        if: matrix.os == 'macos-latest' && matrix.target == 'wasmer_wamr'
-        shell: bash
-        run: brew install cmake ninja
+      - name: install build dependencies (wasmer_wamr)
```

Comment:
Why remove the `macos-latest` check? I see that it's currently not required because the matrix only contains `wasmer_wamr` for `macos-latest` and not Android or other platforms. But the brew command will always be specific to macos. Just a detail though...

Reply:
Because of that, it's only run on macos anyway. But fair enough, it can stay.